test/test.vcxproj

```diff
@@ -25,6 +25,7 @@
   <ImportGroup Label="PropertySheets" />
   <PropertyGroup Label="UserMacros" />
   <ItemGroup>
+    <ClCompile Include="op2ext.test.cpp" />
```

Comment:
I noticed there is no corresponding `test.vcxproj.filters` update for this line.